FLUID_BLOCK.md

```diff
@@ -0,0 +1,312 @@
+# Fluid Block simulation
```

Comment:
the order should be:
1. problem statement
2. the model (classes for block and fluids)
3. How models translate to input files - a bit about the FileManager and the rendering mechanism, without going into many details. Just some snippets to show how object attributes find their way into the files. The user can then go on and inspect deeper
4.  The simulators - talk a bit about the specific implementation details of the simulate method between both cases and how the same models can be used across both scenarios. Here we also talk a bit about the Simulation parameter class to configure the runs. Nothing to extended. The user will get the gist.
5. how to use the simulators and create a few dynamic runs with varying modesl/configs.
6.

Reply:
I think the 2. and 3. points make sense, since looking at it again the templating intro seems cumbersome.

For the 4., I am not sure If we should highlight the construction for both simulators. It seems that we should just point to that.

The 5th wraps up the introduction, so I didn't want to extend to much.